homeworks/01-bubble-sort/src/styles/Footer.scss

```diff
@@ -0,0 +1,76 @@
+footer {
```

Comment:
More preferable to have separate css files for every component. And keep them next to the component file.